internal/pkg/cli/project_delete.go

```diff
@@ -23,8 +23,8 @@ import (
 const (
 	defaultProfile = "default"
 
-	fmtConfirmProjectDeletePrompt = "Are you sure you want to delete project %s?"
-	confirmProjectDeleteHelp      = "Deleting a project will remove all associated resources. (apps, envs, etc.)"
+	fmtConfirmProjectDeletePrompt = "Are you sure you want to delete project %s? This will delete your project as well as any apps, environments, and pipelines."
```

Comment:
How does it look if you move the second sentence to a new line in the CLI, is it more readable? so that the line stays <80 chars
```suggestion
	fmtConfirmProjectDeletePrompt = `Are you sure you want to delete project %s? 
This will delete your project as well as any apps, environments, and pipelines.`
```